app/liquid_tags/link_tag.rb

```diff
@@ -16,7 +17,7 @@ def render(_context)
   end
 
   def get_article(slug)
-    slug = ActionController::Base.helpers.strip_tags(slug).strip
+    slug = strip_tags(slug).strip
```

Comment:
QUESTION: I'm having a little trouble here, and would like some input.

When I run the helper `strip_tags(slug)` on this line, the resulting string has a space inserted at the end.

However, when I run the code that the helper encapsulates, that extra space doesn't get inserted:
```ruby
def strip_tags(string)
    ActionController::Base.helpers.strip_tags(string).strip
end
```
In order to remove that inserted space, I have to call `.strip` on the helper, which makes zero sense to me. I'm not sure why this is happening. I use the `strip_tags` helper elsewhere (eg [HERE](https://github.com/forem/forem/blob/6f8fc9c13d652a7bc8cd8412274db0f7b248360c/app/liquid_tags/youtube_tag.rb#L18)) and this issue does not arise.

Reply:
I set a `binding.pry` breakpoint and ran the specs, here's the slug that comes in: `"/username45/test-this-please-452 "`. As you can see the final space is already part of it and `strip_tags` does not deal with whitespace [1]. So it doesn't really matter which method we are using, the result is the same:

```ruby
strip_tags(slug) == ActionController::Base.helpers.strip_tags(slug)
#=> true
```

In the above diff you can see that the old solution also had an extra `.strip` tucked on. I suggest the following instead:

```ruby
strip_tags(slug.strip)
```

All specs are green with this change.

[1] `strip_tags` generally doesn't quite make sense for me in this context but we had it before so it's fine to keep it.